### DISCUSSION TITLE
v4 版本配置文件中能否嵌入 Shell 语句

### DISCUSSION THREAD

**Discussion by Potat0000** [IrineSistiana/mosdns#436] [Q&A] [answered]
在 v3 版本中可以使用 `${{ cmd }}` 语法在配置文件中运行 Shell 语句。想请问 v4 中有无类似功能？
谢谢！

PS：主要是希望能在 `ecs` 插件中自动填入 IP

**Comment by IrineSistiana** [comment, marked as answer]
不能。也不会再加入。

建议用启动脚本， sed 一把梭。
